locales/en-US/strings.properties

```diff
@@ -136,19 +146,19 @@ pocket_read_even_more=View More Stories
 # LOCALIZATION NOTE (pocket_feedback_header): This is shown as an introduction
 # to Pocket as part of the feedback form.
 pocket_feedback_header=The best of the web, curated by over 25 million people.
-# LOCALIZATION NOTE (pocket_feedback_body): This is shown below
-# (pocket_feedback_header) to provide more information about Pocket.
-pocket_feedback_body=Pocket, a part of the Mozilla family, will help connect you to high-quality content that you may not have found otherwise.
-pocket_send_feedback=Send Feedback
+# LOCALIZATION NOTE (pocket_description): This is shown in the settings pane and
+# below (pocket_feedback_header) to provide more information about Pocket.
+pocket_description=Discover high-quality content you might otherwise miss, with help from Pocket, now part of Mozilla.
 
+highlights_empty_state=Start browsing, and we’ll show some of the great articles, videos, and other pages you’ve recently visited or bookmarked here.
```

Comment:
Unrelated to localization: isn’t *here* way too far from show? It sounds like it’s associated to bookmark (but that might be my non English brain).

Reply:
I believe that "place" adverbs (e.g., here) typically go at the end of sentences even when split by a (long) preposition.

"We'll show some of … here." -> "We'll show some here." vs "We'll show here some."

Or a difference sentence:
"Play with the ball there" vs "Play there with the ball" where the former with "place" adverb at end sounds better (at least to me…)